src/mlx90632.c

```diff
@@ -44,18 +44,25 @@ static const char mlx90632version[] __attribute__((used)) = { VERSION };
 #define STATIC static
 #endif
 
-int mlx90632_start_measurement(void)
+int32_t mlx90632_trigger_measurement(void)
 {
-    int ret, tries = MLX90632_MAX_NUMBER_MESUREMENT_READ_TRIES;
+    int32_t ret;
     uint16_t reg_status;
 
     ret = mlx90632_i2c_read(MLX90632_REG_STATUS, &reg_status);
     if (ret < 0)
         return ret;
 
     ret = mlx90632_i2c_write(MLX90632_REG_STATUS, reg_status & (~MLX90632_STAT_DATA_RDY));
-    if (ret < 0)
-        return ret;
+
+    return ret;
+}
+
+int32_t mlx90632_wait_for_measurement(void)
+{
+    int32_t ret;
+    uint16_t reg_status;
+    int tries = MLX90632_MAX_NUMBER_MESUREMENT_READ_TRIES;
```

Comment:
lets arrange variables in inverted xmass tree:

```suggestion
    int tries = MLX90632_MAX_NUMBER_MESUREMENT_READ_TRIES;
    uint16_t reg_status;
    int32_t ret;
```